.github/workflows/nightly-tests.yml

```diff
@@ -3,10 +3,10 @@ name: Nightly Tests
 on:
   schedule:
     - cron: '0 0 * * *'
-  workflow_dispatch:
 
 jobs:
   tests:
+    if: github.event.ref == 'refs/heads/main'
```

Comment:
This will cause the workflow to run main but will not limit it to the hyperledger fork. See the int test worflow as an example: https://github.com/hyperledger/aries-cloudagent-python/blob/main/.github/workflows/integrationtests.yml#L13